src/cosmosdb-preview/azext_cosmosdb_preview/tests/latest/test_cosmosdb-InAccountRestore_scenario-Table-Gremlin.py

```diff
@@ -0,0 +1,257 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+import unittest
+
+from azure.cli.testsdk.scenario_tests import AllowLargeResponse
+from azure.cli.testsdk import (ScenarioTest, ResourceGroupPreparer)
+from datetime import datetime
+import datetime
+
+TEST_DIR = os.path.abspath(os.path.join(os.path.abspath(__file__), '..'))
+
+class Cosmosdb_previewInAccountRestoreScenarioTest_Table_Gremlin(ScenarioTest):
+
+    @AllowLargeResponse()
+    @ResourceGroupPreparer(name_prefix='cli_test_cosmosdb_gremlin_database')
+    def test_cosmosdb_gremlin_database(self, resource_group):
+        db_name = self.create_random_name(prefix='cli', length=15)
+        location = "WestUS"
+
+        self.kwargs.update({
+            'acc': self.create_random_name(prefix='cli', length=15),
+            'db_name': db_name,
+            'loc': location
+        })
+
+        self.cmd('az cosmosdb create -n {acc} -g {rg} --capabilities EnableGremlin  --backup-policy-type Continuous --locations regionName={loc}')
+
+        assert not self.cmd('az cosmosdb gremlin database exists -g {rg} -a {acc} -n {db_name}').get_output_in_json()
```

Comment:
Instead of calling raw assert, you can leverage `self.cmd('cosmosdb ...', checks=[...])`

Reply:
@ddhamotharan , can you please comment on this change